docs/guides/deploy/ansible.md

```diff
@@ -52,14 +52,14 @@ current directory, and remove the rest of the cloned repository, as it's not req
 
 ```bash
 git clone https://github.com/netdata/community.git
-mv community/configuration-management/ansible .
+mv netdata-agent-deployment/ansible-quickstart .
```

Comment:
```suggestion
mv community/netdata-agent-deployment/ansible-quickstart .
```

Reply:
Good catch, @ilyam8. @voriol can you make this change? Then we'll approve and merge.

Reply:
Thanks for pointing that out @ilyam8. I update the line in patch-1 branch.